### PR TITLE
Flake8 plugins, disable Dependabot labels, fix Labeler

### PR DESCRIPTION
**Pull request by KapJI**
Backport some changes from `ha-google-home`:

- Install Flake8 plugins
- Updated Labeler config with proper globs

Apart from that:

- Fix github-actions updates with Dependabot. I think it doesn't work with `"/.github/workflows"`, at least I haven't seen any PRs for github actions. `"/"` should work for sure.
- I found in [Dependabot sources](https://github.com/dependabot/dependabot-core/blob/2f0db3e851ba2cc43d0b6dcd70da5e69d5b63eb6/common/lib/dependabot/pull_request_creator/labeler.rb#L204) how to disable version labels.

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -56,6 +56,11 @@
   description: Performance
   color: "016175"
 
+# Prevent Dependabot from adding version labels
+- name: skip-release
+  description: Skip release
+  color: "000000"
+
```
Comment (leikoilja) [inline .github/labels.yml:63]: Great find! 
According to their source just by having this label will prevent dependabot from increasing the version, right?

**Review by leikoilja — APPROVED** [review]
Awesome stuff! Thanks

File: .github/labels.yml (hunk 1 above)
Comment (KapJI) [reply .github/labels.yml:63]: [Yep!](https://www.youtube.com/watch?v=sGXQR5Xbpqs)

Comment (leikoilja) [reply .github/labels.yml:63]: haha good one :D

**Pull request merged**